### DISCUSSION TITLE
Add allergies to widget

### DISCUSSION THREAD

**Discussion by galaxyrun** [breezy-weather/breezy-weather#1249] [Ideas]
### Describe your idea

Could you please add allergies to the widget

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.7 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
The keyword for displaying a summary of pollen index is `$0pis$` where `0` is today, `1` tomorrow and so on.

**Reply by galaxyrun** [reply]
Where do I type these in?

**Reply by papjul** [reply]
In the widget settings:
- Subtitle: Custom
- Subtitle data: `$0pis$`. You have the full documentation of all available keywords below.

Not all widgets have this feature, so feel free to check others if you can't find the setting.
